x-pack/plugins/ml/public/application/contexts/kibana/kibana_context.ts

```diff
@@ -12,11 +12,14 @@ import {
 } from '../../../../../../../src/plugins/kibana_react/public';
 import { SecurityPluginSetup } from '../../../../../security/public';
 import { LicenseManagementUIPluginSetup } from '../../../../../license_management/public';
+import { SharePluginStart } from '../../../../../../../src/plugins/share/public';
 
 interface StartPlugins {
   data: DataPublicPluginStart;
   security?: SecurityPluginSetup;
   licenseManagement?: LicenseManagementUIPluginSetup;
+  kibanaVersion: string;
```

Comment:
i don't think `kibanaVersion` should be in here as it's not a plugin.
it could just be added to `StartServices` below.

Reply:
Changed in https://github.com/elastic/kibana/pull/68784/commits/87faf26e372cc8f54dc3ef308672337119d6b2a0